src/cmd-buildupload

```diff
@@ -13,6 +13,12 @@ import tempfile
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 from cmdlib import load_json, Builds  # noqa: E402
 
+# set image artifact caching at 1y; it'll probably get evicted before that...
```

Comment:
Is this based on something like https://stackoverflow.com/questions/2970938/ideal-http-cache-control-headers-for-different-types-of-resources ? If so probably worth referencing.

Reply:
Also on this topic, I wonder if we should have a flag for setting `Cache-Control: public` but, don't need to do that right now.

Reply:
> Is this based on something like https://stackoverflow.com/questions/2970938/ideal-http-cache-control-headers-for-different-types-of-resources ?

Pure fluke (I was initially going to do 10 years). I added a link to that SO though since it actually has an argument.